SU-SWS/stanford_basic
# D8CORE-2592

# READY FOR REVIEW

# Summary
- Remove the duplicate ID's that happens on submenus for the mobile/responsive view.

# Review By (Date)
- Wednesday 10/21

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Create menu links to have sub menus on main pages.
3. change to the mobile view. 
4. Verify that that there are no more id='su-acs-

# Affected Projects or Products
- This is done on Stanford Basic and we also need it on SOE since that is where it was found.

# Associated Issues and/or People
- D8CORE-2592
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
